tests/display.rs

```diff
@@ -350,5 +350,5 @@ fn debug_string() {
     use glium::backend::Facade;
 
     let display = support::build_display();
-    display.get_context().insert_debug_marker("Hello world");
+    display.get_context().insert_debug_marker("Hello world").unwrap();
```

Comment:
What is the test checking? Should there be another test to see whether `insert_debug_marker` works on the given platform? Maybe we can have some kind of conditional that expects failure on certain platforms?